### PR TITLE
Update tutorial links to point to README

### PR DESCRIPTION
**Pull request by emlaver**
- Updated spark link to open README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,5 +19,9 @@ For example care should be taken around handling:
 * nested JSON structures and other data type limitations
 
 1. [IBM Cloud Functions](./cloud-functions)
-2. [Apache Spark on Data Science Experience](./spark-on-dsx)
-2. [Node-RED on IBM Cloud](./node-red)
+1. [Apache Spark on Data Science Experience](./spark-on-dsx/README.md)
+
+    For the Apache Spark™/Bahir sql-cloudant connector solution, customers are responsible for creating and
```
Comment (ricellis) [inline README.md]: Wouldn't this  be better placed in the Apache Spark specific README at https://github.com/cloudant-labs/data-flow-examples/blob/master/spark-on-dsx/README.md ?

Comment (emlaver) [reply README.md]: I've rebased and removed the note as I don't think it adds much value.

File: README.md [hunk 2]
```diff
@@ -19,5 +19,5 @@ For example care should be taken around handling:
 * nested JSON structures and other data type limitations
 
 1. [IBM Cloud Functions](./cloud-functions)
-2. [Apache Spark on Data Science Experience](./spark-on-dsx)
-2. [Node-RED on IBM Cloud](./node-red)
+1. [Apache Spark on Data Science Experience](./spark-on-dsx/README.md)
```
Comment (ricellis) [inline README.md]: Add links to README file for other items too

**Review by ricellis — APPROVED** [review]
+1

File: README.md (hunk 2 above)
Comment (emlaver) [reply README.md]: Updated in 1fcb4ed.

**Pull request merged**